File: src/main/java/com/vocalclerk/api/dtos/JwtAuthResponse.java
package com.vocalclerk.api.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * DTO for JWT authentication response.
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class JwtAuthResponse {
    
    private String accessToken;
    private String tokenType = "Bearer";
    private long expiresIn;
    private UserDto user;
}

File: backend/DoNow.Application/DTOs/JwtAuthResponse.cs
namespace DoNow.Application.DTOs;

public class JwtAuthResponse
{
    public string AccessToken { get; set; } = string.Empty;
    public string TokenType { get; set; } = "Bearer";
    public long ExpiresIn { get; set; }
    public UserDto User { get; set; } = null!;
}
